ocrd_network/ocrd_network/deployer.py

```diff
@@ -278,17 +279,23 @@ def start_native_processor(self, client: SSHClient, processor_name: str, queue_u
         Returns:
             str: pid of running process
         """
-        # TODO: some processors are bashlib. They have to be started differently
         self.log.info(f'Starting native processor: {processor_name}')
         channel = client.invoke_shell()
         stdin, stdout = channel.makefile('wb'), channel.makefile('rb')
-        cmd = f'{processor_name} --database {database_url} --queue {queue_url}'
+        if is_bashlib_processor(processor_name):
+            cmd = f'ocrd processing-worker {processor_name} --database {database_url} ' \
+                f'--queue {queue_url}'
```

Comment:
Not a good pattern IMO. Instead of the heuristic to determine whether a processor is Pythonic or not, why not just **implement** the `--database ... --queue` options in bashlib, as it should be?

Reply:
@joschrew, could you check this? Or @bertsky, could you provide more context on how you expect this to be implemented?

Reply:
Well, among the recent changes @joschrew introduced `is_bashlib_processor`. Instead of trying to look inside – which is error-prone, and by OCR-D CLI we might even deal with pure program code (binaries), which would not work at all – I recommend simply adding the Processing Server functionality to bashlib's `ocrd__wrap`, so bashlib-based processors behave like Pythonic processors. Of course, this would simply delegate to `ocrd processing-worker` internally.

Reply:
Let's continue discussion in https://github.com/OCR-D/core/issues/1023

Reply:
@kba Since this is trivial to implement in lib.bash, and error-prone here, I recommend unresolving, and doing it right in the first place.

Reply:
Resolved after merging #1024